woocommerce/woocommerce-android
# Issue/5112 quick order view

Closes #5112 - This PR introduces the "quick order" screen and related view model. No order is created yet, this is simply the UI for it.

To test:

* Make sure the Quick Order beta feature is enabled
* Go to the order list
* Tap the + icon
* Verify that the quick order dialog shows the correct currency
* Switch to another store that uses a different currency
* Verify that the quick order dialog shows the different currency
* Verify that the dialog appears correctly in both light and dark modes

![us](https://user-images.githubusercontent.com/3903757/140559763-ab60a4a2-a1b3-4ab0-b982-b064419a6c28.png)

![nonus](https://user-images.githubusercontent.com/3903757/140559785-6182eb1d-7216-461c-aa84-f5aa97828128.png)

![dark](https://user-images.githubusercontent.com/3903757/140559796-2d34b47d-5eb7-400c-8992-caff741c689f.png)
